package.json

```diff
@@ -34,7 +34,7 @@
     "prebuild": "rimraf build",
     "build": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json && echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
     "build:watch": "tsc -b --watch ./tsconfig.cjs.json ./tsconfig.esm.json && echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
-    "lint": "eslint ./lib ./test --ext ts",
+    "lint": "eslint .",
```

Comment:
Instead of only linting the source and tests, this now lints all files within the project.